packages/removable-tags/src/components/RemovableTags.test.js

```diff
@@ -0,0 +1,17 @@
+import React from "react";
+import Enzyme, { shallow } from "enzyme";
+import Adapter from "enzyme-adapter-react-16";
+
+import RemovableTags from "./RemovableTags";
+
+Enzyme.configure({ adapter: new Adapter() });
+
+describe("Removable tags", function() {
+  test("should call onClose when clicking on the icon", function() {
+    const onClickMock = jest.fn();
+    const wrapper = shallow(<RemovableTags onClose={onClickMock} />);
```

Comment:
Minor: The function name is `onClickMock` but you are mocking a `onClose`. The naming should be consistent.